### PR TITLE
Add ellipsis to assistant picker + wording and spacing changes

### PR DESCRIPTION
**Pull request by albandum**
## Description

- Add an ellipsis link to the Assistants picker list 
- Edit broken links

Screenshot 2024-01-24 at 14 25 06

## Risk

None

### CODE REVIEW

File: sparkle/src/components/Item.tsx [hunk 1]
```diff
@@ -96,72 +100,95 @@ export function Item({
...
+                visual={More}
+                className={classNames(
+                    "s-shrink-0 s-transition-all s-duration-200 s-ease-out s-opacity-50 hover:s-opacity-100 cursor-pointer",
+                    disabled
+                      ? "s-text-element-500 dark:s-text-element-500-dark"
+                      : selected
+                      ? "s-text-action-400 s-opacity-100 dark:s-text-action-600-dark"
+                      : classNames(
+                          "s-text-element-600 group-hover:s-text-action-400 group-active:s-text-action-700 dark:group-hover:s-text-action-600-dark dark:group-active:s-text-action-400-dark",
+                          hasAction ? "group-hover:s-opacity-100" : ""
```
Comment (spolu) [inline sparkle/src/components/Item.tsx:183]: hasAction?

**Review by spolu — COMMENTED** [review]
Let's remove onUpdate from assistantDetails and move the call to useAgentConfigurations to get mutateAgentConfigurations inside AssistantDetails to avoid making too many queries to useAgentConfiguraions (here whenever we show the AssistantPicker)

File: front/components/assistant/AssistantPicker.tsx [hunk 2]
```diff
@@ -17,6 +19,8 @@ import { useEffect, useState } from "react";
 
 import { filterAndSortAgents } from "@app/lib/utils";
 
+import { AssistantDetails } from "./AssistantDetails";
```
Comment (spolu) [inline front/components/assistant/AssistantPicker.tsx:22]: Please update to @app based imports :pray:

**Review by spolu — APPROVED** [review]
Minor nit on import path. Otherwise LGTM

Feel free to merge once addressed!

File: front/components/assistant/AssistantPicker.tsx [hunk 3]
```diff
@@ -53,59 +69,78 @@ export function AssistantPicker({
...
+                    }
+                  }}
+                />
+              </div>
+            )}
+          </>
+        }
+        bottomBar={
+          <div className="flex border-t border-structure-50 p-2">
             <Link href={`/w/${owner.sId}/builder/assistants/new`}>
```
Comment (spolu) [inline front/components/assistant/AssistantPicker.tsx:98]: You lost the flow diff?

Comment (albandum) [reply front/components/assistant/AssistantPicker.tsx:97]: What do you mean by the flow diff ?

**Review by spolu — COMMENTED** [review]
I think we lost an update to `/w/${owner.sId}/builder/assistants/new` (adding the flow argument)? Otherwise LGTM!

**Review by spolu — APPROVED** [review]

**Pull request merged**